octoprint_slic3r/__init__.py

```diff
@@ -166,7 +166,7 @@ def get_slicer_properties(self):
 	def get_slicer_default_profile(self):
 		path = self._settings.get(["default_profile"])
 		if not path:
-			path = os.path.join(os.path.dirname(os.path.realpath(__file__)), "profiles", "default.profile.ini")
+			path = os.path.join(os.path.dirname(os.path.realpath(__file__)), "static/profiles", "default.profile.ini")
```

Comment:
Honestly, I'm not sure which way is it better. I've just did it as I've seen it in M3D-Fio. But when you ask, maybe better way will be to just let it be in profiles and add `profiles` folder here https://github.com/Javierma/OctoPrint-Slic3r/blob/master/setup.py#L27
If you think, the "profiles" folder way is better, I'll just change it in my repo and when I push changes. you'll see them right here.